test/load/test_load_tx_ethbridge_burn.py

```diff
@@ -22,30 +23,97 @@
 # E0326 11:41:27.125006742  636480 fork_posix.cc:70]           Fork support is only compatible with the epoll1 and poll polling strategies
 # Maybe: https://github.com/grpc/grpc/issues/29044
 
+eth_account_number = 2
+sif_account_number = 1
+transaction_number = 2
+
+# build the transfer matrix 
+def build_transfer_table() -> [[int]]:
+    transfer_table = [[transaction_number for y in range(eth_account_number)] for x in range(sif_account_number)]
+    return transfer_table
+
+# create an erc20 contract for rowan via lock rowan in sifnode
+def burn_rowan_get_erc20_address(ctx: test_utils.EnvCtx) -> Address:
+    # check contract not created before
+    token_address = ctx.get_destination_contract_address(rowan)
+    if token_address != NULL_ADDRESS:
+        return token_address
+    fund_amount_sif = 10 ** 20
+    fund_amount_eth = 10 ** 20
+    rowan_transfer_amount = 10 ** 18
+    test_sif_account = ctx.create_sifchain_addr(fund_amounts={rowan:fund_amount_sif, ctx.ceth_symbol:fund_amount_eth})
+    test_eth_account = ctx.create_and_fund_eth_account(fund_amount=fund_amount_eth)
+    ctx.sifnode_client.send_from_sifchain_to_ethereum(test_sif_account, test_eth_account, rowan_transfer_amount, rowan)
+    ctx.advance_blocks()
+
+    # wait for erc20 contract created
+    token_address = ctx.wait_for_new_bridge_token_created(rowan)
+    return token_address
+
+# test single sif account burn erc20 to multiple ethereum accounts
+def test_single_sif_to_multiple_eth_account_lock_rowan(ctx: test_utils.EnvCtx):
+    # get rowan contract address
+    rowan_token_address = burn_rowan_get_erc20_address(ctx)
+
+    print("+++++++++++++++++++++++++ ", rowan_token_address)
```

Comment:
This probably needs to be cleaned up? If you want to keep it, it's better to use logging.